utils/utils.go

```diff
@@ -585,3 +590,141 @@ func GetWLOLicenseAnnotations() map[string]string {
 	annotations["productChargedContainers"] = "ALL"
 	return annotations
 }
+
+func isVolumeMountFound(pts *corev1.PodTemplateSpec, name string) bool {
+	for _, v := range pts.Spec.Containers[0].VolumeMounts {
+		if v.Name == name {
+			return true
+		}
+	}
+	return false
+}
+
+func isVolumeFound(pts *corev1.PodTemplateSpec, name string) bool {
+	for _, v := range pts.Spec.Volumes {
+		if v.Name == name {
+			return true
+		}
+	}
+	return false
+}
+
+// ConfigureLTPA setups the shared-storage for LTPA keys file generation
+func ConfigureLTPA(pts *corev1.PodTemplateSpec, la *wlv1.WebSphereLibertyApplication, operatorShortName string) {
+	// Create an emptyDir volume at /config/configDropins/overrides
+	emptyDirLtpaServerXMLVolumeMount := GetEmptyDirLTPAServerXMLVolumeMount(la)
+	if !isVolumeMountFound(pts, emptyDirLtpaServerXMLVolumeMount.Name) {
+		pts.Spec.Containers[0].VolumeMounts = append(pts.Spec.Containers[0].VolumeMounts, emptyDirLtpaServerXMLVolumeMount)
+	}
+	if !isVolumeFound(pts, emptyDirLtpaServerXMLVolumeMount.Name) {
+		vol := corev1.Volume{
+			Name: emptyDirLtpaServerXMLVolumeMount.Name,
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		}
+		pts.Spec.Volumes = append(pts.Spec.Volumes, vol)
+	}
+
+	// Mount a volume /config/ltpa/keys to store the ltpa.keys file
+	ltpaKeyVolumeMount := GetLTPAVolumeMount(la, "keys")
+	if !isVolumeMountFound(pts, ltpaKeyVolumeMount.Name) {
+		pts.Spec.Containers[0].VolumeMounts = append(pts.Spec.Containers[0].VolumeMounts, ltpaKeyVolumeMount)
+	}
+	if !isVolumeFound(pts, ltpaKeyVolumeMount.Name) {
+		vol := corev1.Volume{
+			Name: ltpaKeyVolumeMount.Name,
+			VolumeSource: corev1.VolumeSource{
+				Secret: &corev1.SecretVolumeSource{
+					SecretName: operatorShortName + "-managed-ltpa",
+				},
+			},
+		}
+		pts.Spec.Volumes = append(pts.Spec.Volumes, vol)
+	}
+
+	// Mount a volume /config/ltpa/xml to store the Liberty Server XML (ltpa.xml)
+	ltpaXMLVolumeMount := GetLTPAVolumeMount(la, "xml")
+	if !isVolumeMountFound(pts, ltpaXMLVolumeMount.Name) {
+		pts.Spec.Containers[0].VolumeMounts = append(pts.Spec.Containers[0].VolumeMounts, ltpaXMLVolumeMount)
+	}
+	if !isVolumeFound(pts, ltpaXMLVolumeMount.Name) {
+		vol := corev1.Volume{
+			Name: ltpaXMLVolumeMount.Name,
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: operatorShortName + LTPAServerXMLSuffix,
+					},
+				},
+			},
+		}
+		pts.Spec.Volumes = append(pts.Spec.Volumes, vol)
+	}
+
+	// Create an initContainer to copy /config/ltpa/xml/ltpa.xml into the emptyDir volume at /config/configDropins/overrides
```

Comment:
@kabicin please try the `subPath` approach (mentioned in the articles [here](https://imantumorang.com/posts/mount-file-to-kubernetes-pod-without-deleting-the-existing-file-in-the-docker-container-in-the-same-directory/) and [here](https://www.golinuxcloud.com/mount-configmap-as-file-in-existing-directory/)) to include the LTPA config into an existing directory with files. Assuming it achieves the same result, I think this approach would be better than using the init container (the primary/app container is not started until init containers start and complete their process. Not a significant delay, but still some delay):

Reply:
I was able to use subPath so I removed the initContainer